crosscheckFingerprintsCollector.wdl

```diff
@@ -128,8 +175,8 @@ workflow crosscheckFingerprintsCollector {
      }
 
     meta {
-     author: "Lawrence Heisler"
-     email: "lawrence.heisler@oicr.on.ca"
+     author: "Lawrence Heisler and Gavin Peng"
+     email: "lawrence.heisler@oicr.on.ca and gpeng@oicr.on.ca"
      description: "crosscheckFingerprintsCollector, workflow that generates genotype fingerprints using gatk ExtractFingprint.  Output are vcf files that can be proccessed through gatk Crosscheck fingerprints\n##"
      dependencies: [
```

Comment:
add modules to the dependencies, see the mavis workflow as an example

then regenerate the readme so that these appear

Reply:
dependency and readme updated.